src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -207,7 +217,8 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
           </Grid>
           <Grid item xs={10}>
             <Link component={RouterLink} to={`${routes.PROJECTS}/${wbsPipe(changeRequest.wbsNum)}`}>
-              {wbsPipe(changeRequest.wbsNum)}
+              {wbsPipe(changeRequest.wbsNum)} - {projectName ? projectName : ''}
```

Comment:
Also, this is my mistake, but can we replace `projectName ? projectName : ''` to just `projectName` since passing line 103 indicates that projectName is defined and it's a string type?